src/components/ActNowCard/BaseContent.js

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import { CardContent, Typography, Grid } from "@material-ui/core";
+import CloseIcon from "@material-ui/icons/Close";
+
+import useStyles from "./useStyles";
+
+const BaseCard = ({ close, onCloseCard, title, description, children }) => {
+  const classes = useStyles();
+
+  return (
+    <CardContent className={classes.cardContent}>
+      {close && (
+        <CloseIcon onClick={onCloseCard} className={classes.closeIcon} />
+      )}
+      {title && (
+        <Grid
+          container
+          justify="center"
+          className={!close ? classes.titleContent : ""}
+        >
+          <Typography variant="h4">{title}</Typography>
+        </Grid>
+      )}
```

Comment:
```suggestion
        <Typography align="center" variant="h4">{title}</Typography>
      )}
```

If all we need is centring, I think 👆🏾 should do it.